src/schema/v2/homeView/sections.ts

```diff
@@ -36,6 +36,7 @@ export type HomeViewSection = {
     href?: MaybeResolved<string>
     behaviors?: HomeViewComponentBehaviors
   }
+  requiresAuthentication: boolean
```

Comment:
`HomeViewSection` declarations now have this additional attribute, which helps the zone logic determine whether they should be included.